src/app/model-download/components/model-download/model-download.component.html

```diff
@@ -0,0 +1 @@
+<kern-model-download-component></kern-model-download-component>
```

Comment:
i think the file path might be misleading, this is placed in model-download/components but it's the page, right?

Reply:
Added it in a folder pages and the component stays in a folder components